src/api/color.rs

```diff
@@ -41,33 +41,34 @@ pub use v_frame::pixel::ChromaSampling;
 /// Supported Color Primaries
 ///
 /// As defined by “Color primaries” section of ISO/IEC 23091-4/ITU-T H.273
+#[wasm_bindgen]
 #[derive(
   ArgEnum, Debug, Clone, Copy, PartialEq, FromPrimitive, Serialize, Deserialize,
 )]
 #[repr(C)]
 pub enum ColorPrimaries {
+  /// Unspecified, must be signaled or inferred outside of the bitstream
+  Unspecified = 0,
   /// BT.709
   BT709 = 1,
-  /// Unspecified, must be signaled or inferred outside of the bitstream
-  Unspecified,
-  /// BT.470 System M (historical)
-  BT470M = 4,
   /// BT.470 System B, G (historical)
-  BT470BG,
+  BT470BG = 2,
```

Comment:
AFAIK these numbers are defined as per spec `ISO/IEC 23091-4/ITU-T H.273`
Here 2 is unspecified, 3 is reserved so you cannot keep numbers like that IIUC.

Reply:
Okay. That is basically what my question is about.

The problem is, that I either need to specify no or all options in the enum with a number.
I will have a look into the spec.